.storybook/playground-helpers.ts

```diff
@@ -0,0 +1,27 @@
+export const jsx = `<div className="container">
+  <div>
+    <VibeNext.Heading>Online Playground</VibeNext.Heading>
+    <VibeNext.Heading type="h3" color="secondary">
+      Prototype with actual components
+    </VibeNext.Heading>
+  </div>
+  <div>
+    <Text>
+      Can't see the code editor?
+      <br />
+      Click on the <VibeIcons.Settings /> button on the left panel and select
+      the <b>"Change Addons Orientation"</b>{" "}
```

Comment:
Maybe use Text with weight? To leverage the Typography component

Reply:
Done 💪🏼